src/logging/logging.module.ts

```diff
@@ -36,7 +36,7 @@ function winstonTransportsFactory(
     level: configurationService.getOrThrow<string>('log.level'),
     format: prettyColorize
       ? winston.format.prettyPrint({ colorize: true })
-      : winston.format.json(),
+      : winston.format.combine(winston.format.errors(), winston.format.json()),
```

Comment:
It is also [possible to access the stack](https://github.com/winstonjs/logform?tab=readme-ov-file#errors) of an `Error` should we want to in the future.